crypto/blake2/blake2.c

```diff
@@ -158,7 +168,7 @@ void BLAKE2B256_Final(uint8_t out[BLAKE2B256_DIGEST_LENGTH], BLAKE2B_CTX *b2b) {
   blake2b_transform(b2b, b2b->block, b2b->block_used,
                     /*is_final_block=*/1);
   OPENSSL_STATIC_ASSERT(BLAKE2B256_DIGEST_LENGTH <= sizeof(b2b->h), _)
-  memcpy(out, b2b->h, BLAKE2B256_DIGEST_LENGTH);
+  copy_digest_words_to_dest(out, b2b->h, BLAKE2B256_DIGEST_LENGTH / 8);
```

Comment:
`BLAKE2B256_DIGEST_LENGTH` is defined in bytes, so we're assuming 8-byte words here. can we count on words always being 8 bytes?